### PR TITLE
Add FAQ entry on installing language packs offline.

### LINKED ISSUE #4049: Document the ability to install language packs offline
(From call with Team4Tech today)

Right now, [according to our documentation](http://field.learningequality.org/develop/docs/usermanual/userman_admin.html#language-packs), you need to have Internet access to download language packs. However, we actually have a management command (`kalite manage languagepackdownload --from-file=FILE`) that allows for this to be done offline, and I was asked today how to do that.

Two things are needed here:
- Additional documentation, possibly as a separate "doing installations offline" section also linked from both the Download Videos and the Download Language Packs sections. Basically, we need to tell people what files to copy, where to put the files after installing, and what commands to run.
- Links to the zip files for the language packs for each language. Might be easiest to just hardcode these into the docs for now, unless we want to finish fixing up https://hub.learningequality.org/languages/dashboard instead (@aronasorman?)

This isn't a release blocker, but will be important to do by late July for Team4Tech's deployments.

### PR DESCRIPTION
**Pull request by aronasorman**
Fixes #4049.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -35,6 +35,50 @@ I would like to download the videos for KA Lite via BitTorrent, is this possible
...
+
+where <code> is replaced by the right language code. You can now carry around that zip file to computers you want to install the language pack to.
+
+You can install the language pack with the following command (for Debian and OS X)::
+
+  kalite manage languagepackdownload --from-file=<path to zip file> -l<language code> --commandline
+
+Or for Windows::
+
+  <path to KA Lite installation>\bin\windows\kalite.bat languagepackdownload --from-file=<path to zip file> -l<language code> --commandline
```
Comment (MCGallaspy) [inline docs/faq.rst]: Is this interactive? `kalite.bat` currently has a known issue where it can't properly handle user input.

Comment (aronasorman) [inline docs/faq.rst]: Not interactive, the `--commandline` flag is for just allowing us to run this thing in the terminal.

**Comment by MCGallaspy** [comment]
Whenever you give an example like this: `<path to KA Lite installation>\bin\windows\kalite.bat languagepackdownload --from-file=<path to zip file> -l<language code> --commandline`

Where the parts enclosed in the angle brackets (`<` & `>`) are meant to be replaced, I would also provide an example after the replacement is made -- the notation be unfamiliar to some people. (I can dig up some old emails where this was the case, if you want some proof!)

**Comment by aronasorman** [comment]
I concur. Let me add that.

File: docs/faq.rst [hunk 2]
```diff
@@ -35,6 +35,54 @@ I would like to download the videos for KA Lite via BitTorrent, is this possible
...
+
+An example invocation for Windows would be::
+
+  C:\Program Files\KA Lite\ka-lite\bin\windows\kalite.bat languagepackdownload --from-file="C:\Downloads\es.zip" -les --commandline
```
Comment (aronasorman) [inline docs/faq.rst]: @MCGallaspy example here.

**Pull request merged**